src/main/java/lotto/ui/OutputView.java

```diff
@@ -4,11 +4,21 @@
 import lotto.LottoResultManager;
 import lotto.domain.LottoPrice;
 import lotto.domain.LottoTicket;
+import lotto.domain.LottoTicketCollection;
 
 public class OutputView {
 
-    public void displayLottoTickets(List<LottoTicket> tickets) {
-        System.out.println(tickets.size() + "개를 구매했습니다.");
+    public void displayLottoTickets(LottoTicketCollection ticketCollection) {
+        System.out.printf("수동으로 %d장, 자동으로 %d개를 구매했습니다. \n",
+            ticketCollection.getManualTickets().size(), ticketCollection.getAutoTickets().size());
+
+        for (List<LottoTicket> tickets : ticketCollection.getAllTicketCollection()) {
+            displayTicketNumbers(tickets);
+        }
```

Comment:
```suggestion
        for (LottoTicket tickets : ticketCollection.getAllTicketCollection()) {
            displayTicketNumbers(ticket);
        }
```
리스트로 묶어서 처리하면 이렇게 처리할 수 있습니다.